### DISCUSSION TITLE
ソフトウェアエンジニアリング(工学)として InnerSource を位置づける

### DISCUSSION THREAD

**Discussion by masskaneko** [InnerSourceCommons/jp-roadmap#26] [General]
SWEBOK, PMBOK, SQuBOK ほかソフトウェアエンジニアリング体系を語る書籍にはインナーソースが載っていません。
工学的な位置づけを探究すれば以下のように進めるでしょう。

- 近い技術や流行と関連づけて覚えやすくなる (例：継続的インテグレーション)
- 学術的な場で紹介できる (例：情報処理学会)

そのための活動としては以下が挙げられます。

- インナーソースを扱っている論文を調査する
    - ここ20年でのソフトウェアエンジニアリングの国際学会では20本ほど出てきました
    - 和文でもあるかもしれません
- [情報処理学会デジタルプラクティス](https://www.ipsj.or.jp/dp/index.html)に応募する
    - デジタルプラクティスという産業界の取り組みを紹介する枠があります。必ずしも技術的な新規性を示さなくてもよい枠です。
    - ISC Japan による登壇や翻訳が何本か出てきて、上記の論文調査で結果が出たら可能性があります
- [情報処理学会 ソフトウェア工学研究会 ウィンターワークショップ](https://wws.sigse.jp/2022/) に応募する
    - 集中的に議論したいトピックを持ち寄ってグループディスカッションする場があります。わりと学術に近い人が集まりますので、そちらの方々に知ってもらうのによさそうです。
    - ISC Japan による登壇や翻訳が何本か出てきて、上記の論文調査で結果が出たら可能性があります
- [SWEBOK V4 public review](https://www.computer.org/volunteering/boards-and-committees/professional-educational-activities/software-engineering-committee/swebok-evolution) でインナーソースを入れられるか検討する
    - SWEBOK はソフトウェア工学知識体系を書籍化したもので、ACM, IEEE が中心となって進めています。いま、その最新版である V4 のpublic review が進んでいます。
    - 同じく、ISC Japan による登壇や翻訳が何本か出てきて、上記の論文調査で結果が出たら可能性があります

**Comment by yuhattor** [comment]
重要だと思います。
ここで活動している人に対して、インナーソースを売り込んで InnerSource Commons に興味を持ってもらうチャンスですよね。
たぶん違う言葉で InnerSource について語っていることが多いのかなと思います
